src/Umbraco.Cms.Integrations.DAM.Aprimo/Services/IAprimoService.cs

```diff
@@ -4,7 +4,9 @@ namespace Umbraco.Cms.Integrations.DAM.Aprimo.Services
 {
     public interface IAprimoService
     {
-        Task<AprimoResponse<SearchedRecordsPaged>> SearchRecords(string page, string searchTerm);
+        Task<AprimoResponse<SearchItemsPaged<Language>>> GetLanguages();
```

Comment:
Maybe by adding async and sync methods here - e.g. `GetLanguages()` and `GetLanguagesAsync()`, `SearchRecords()` and `SearchRecordsAsync()` - you could make it a bit easier to use for calling code to select the right one?  Then you just have one place to remember to do all the `ConfigureAwait` /  `GetAwaiter().GetResult()` pieces.